README.md

```diff
@@ -131,55 +131,62 @@ What above config means is for eg: partial `yingyang` will be replaced with part
 Similar syntax applies to the rest of the config.
 To learn more about semantics of the config and other options available, please be sure to read docs [here](http://github.com/krakenjs/karka).
 
-* Modify `config/config.json` to load in these rules under the property `specialization`
-
-```
-"specialization": "import:./specialization.json"
-```
-
 ### Including specialization into the render work flow
-#### Modify `view engines` in `config/config.json`
+
+#### Modify `view engines` in `config/config.json` to use `makara@2` to use `makara@2` to use `makara@2` to use `makara@2`
```

Comment:
+#### Modify `view engines` in `config/config.json` to use `makara@2` to use `makara@2` to use `makara@2` to use `makara@2`